src/Blueprint/Blueprint/Models/PSBlueprintAssignment.cs

```diff
@@ -132,5 +126,33 @@ internal static PSBlueprintAssignment FromAssignment(Assignment assignment, stri
 
             return psAssignment;
         }
+
+        private static PSParameterValueBase GetAssignmentParameters(KeyValuePair<string, ParameterValueBase> parameterKvp)
+        {
+            PSParameterValueBase parameter = null;
+
+            if (parameterKvp.Value != null && parameterKvp.Value.GetType().Equals(typeof(ParameterValue)))
+            {
+                // Need to cast as ParameterValue since assignment.Parameters value type is ParamaterValueBase. 
+                var paramaterValue = parameterKvp.Value as ParameterValue;
+
+                parameter = new PSParameterValue { Description = paramaterValue.Description, Value = paramaterValue.Value };
+            }
+            else if (parameterKvp.Value != null && parameterKvp.Value.GetType().Equals(typeof(SecretReferenceParameterValue)))
```

Comment:
@filizt same comment here -- `parameterKvp.Value is SecretReferenceParameterValue`

Reply:
Done.